DigitalOffice/Tests/FileServiceTests/Mappers/FileMapperTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using FileService.Database.Entities;
+using FileService.Mappers;
+using FileService.Mappers.Interfaces;
+using FileService.Models;
+using NUnit.Framework;
+using System;
+
+namespace FileServiceUnitTests.Mappers
+{
+    public class FileMapperTests
+    {
+        private DbFile dbFile;
+        private FileRequest fileRequest;
+        private IMapper<DbFile, FileRequest> dbToRequestMapper;
+        private IMapper<FileRequest, DbFile> requestToDbMapper;
+
+        [SetUp]
+        public void Initialization()
+        {
+            dbToRequestMapper = new FileMapper();
+            requestToDbMapper = new FileMapper();
+
+            fileRequest = new FileRequest
+            {
+                Content = "RGlnaXRhbCBPZmA5Y2U=",
+                Extension = ".txt",
+                Name = "DigitalOfficeTestFile"
+            };
+
+            dbFile = new DbFile
+            {
+                Id = Guid.NewGuid(),
+                Content = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U="),
+                Extension = ".txt",
+                IsActive = true,
+                Name = "DigitalOfficeTestFile"
+            };
+        }
+
+        [Test]
+        public void FailedMappingObjectIsNullTest()
+        {
+            fileRequest = null;
+            dbFile = null;
+
+            Assert.Throws<NullReferenceException>(() => dbToRequestMapper.Map(dbFile));
```

Comment:
It is bad practice to test two different methods in the same test.

Reply:
Fixed